[DOCS] Add tip for Kibana Fleet APIs (#75711)
elastic/elasticsearch: docs/reference/fleet/index.asciidoc

[role="xpack"]
[[fleet-apis]]

> [TIP]
> ====
> If you are looking for the {kib} {fleet} API Documentation, please see the 
> {ref}/fleet/{branch}/fleet-api-docs.html[`Fleet API Documentation`].
> ====

Reviewer: ```suggestion

TIP: For the {kib} {fleet} APIs, see the
{fleet-guide}/fleet-api-docs.html[`Fleet API Documentation`].
```
